crates/jstzd/src/task/async_octez_node.rs

```diff
@@ -8,7 +8,7 @@ use tokio::process::{Child, Command};
 
 use anyhow::{anyhow, Result};
 
-fn path_or_default<'a>(path: Option<&'a PathBuf>, default: &'a str) -> &'a str {
+pub fn path_or_default<'a>(path: Option<&'a PathBuf>, default: &'a str) -> &'a str {
```

Comment:
need this for the following pr

Reply:
I suggest you add it in the following PR